Makefile.am

```diff
@@ -807,8 +812,23 @@ test_serializers_SOURCES = test/serializers.cc test/test.cc
 test_serializers_CPPFLAGS = $(DEPS_CFLAGS) $(SERVICE_DEPS_CFLAGS) @BOOST_CPPFLAGS@ @RAPIDJSON_CPPFLAGS@
 test_serializers_LDADD = $(DEPS_LIBS) $(SERVICE_DEPS_LIBS) @BOOST_LDFLAGS@ $(BOOST_LIBS) libvalhalla.la
 test_traffic_matcher_SOURCES = test/traffic_matcher.cc test/test.cc
-test_traffic_matcher_CPPFLAGS = $(DEPS_CFLAGS) $(SERVICE_DEPS_CFLAGS) @BOOST_CPPFLAGS@
-test_traffic_matcher_LDADD = $(DEPS_LIBS) $(SERVICE_DEPS_LIBS) @BOOST_LDFLAGS@ $(BOOST_LIBS) libvalhalla.la
+test_traffic_matcher_CPPFLAGS = $(DEPS_CFLAGS) @BOOST_CPPFLAGS@
```

Comment:
just in case youll want the `service_deps_cflags` and `libs`. currently these things probably dont make use of them but it might be intractable to manage havingg it on some and not on others